src/pages/Character/CharacterSheetPage/noteLayout/CharacterSidebar/TracksPanel/ProgressTrack.tsx

```diff
@@ -171,7 +176,7 @@ export function ProgressTrack(props: ProgressTracksProps) {
       rollTrackProgress(
         trackType,
         label || "",
-        Math.min(Math.floor(value / 4), 10),
+        useMaxRoll ? 10 : Math.min(Math.floor(value / 4), 10),
         move?._id ?? ""
       );
     }
```

Comment:
This logic might need to get looked at? Or maybe this old character is just bugged lol

![image](https://github.com/user-attachments/assets/209ba83a-8d3c-4c8e-b16e-d3f78c5aee67)

Reply:
Lol nevermind - my brain was confusing bonds and discoveries. :facepalm: